po/pt.po

```diff
@@ -43,36 +43,32 @@ msgid "Monitor"
 msgstr "Monitor"
 
 #: src/Widgets/Headerbar.vala:23
-#, fuzzy
 msgid "End Process"
 msgstr "Terminar processo"
 
 #: src/Widgets/Headerbar.vala:26
-#, fuzzy
 msgid "End selected process"
-msgstr "Terminar processo"
+msgstr "Terminar processo selecionado"
 
 #: src/Widgets/Headerbar.vala:30
-#, fuzzy
 msgid "Kill Process"
-msgstr ""
+msgstr "Matar processo"
 
 #: src/Widgets/Headerbar.vala:32
 msgid "Kill selected process"
-msgstr ""
+msgstr "Matar processo selecionado"
 
 #: src/Widgets/Headerbar.vala:42
 msgid "Settings"
-msgstr "Preferências"
+msgstr "Definições"
 
 #: src/Widgets/Headerbar.vala:56
-#, fuzzy
 msgid "Show an indicator:"
-msgstr "Mostrar ícone do painel"
+msgstr "Mostra um indicador:"
 
 #: src/Widgets/Headerbar.vala:62
 msgid "Start in background:"
-msgstr ""
+msgstr "Inicia em segundo plano"
```

Comment:
I don't understand Portuguese, but is it intentional that the trailing colon is missing?

Reply:
My mistake. I forgot to put the colon. (I did not validate when editing the the po file in Poedit so I didn't see that mistake).
Done